viper/src/verifier.rs

```diff
@@ -80,7 +121,7 @@ impl<'a> Verifier<'a> {
         match backend {
             VerificationBackend::Silicon => silicon::Silicon::with(env).new(reporter, debug_info),
             VerificationBackend::Carbon => {
-                carbon::CarbonVerifier::with(env).new(reporter, debug_info)
+                panic!("Trying to use Carbon backend - now made compatible just with Silicon");
```

Comment:
The subject of "now made compatible just with Silicon" appears to be Carbon, but should it be something like "the viper crate"?

Reply:
It would be useful to add a comment before the panic that explains why Carbon is no longer supported, so that we know what needs to be fixed.